models/models.js

```diff
@@ -3,32 +3,55 @@ const db = require("../db/connection.js");
 
 function selectTopics () {
   return db.query(`SELECT * FROM topics;`).then((result) => {
-    return result.rows;
-  });
-};
+    return result.rows
+  })
+}
 
 function selectArticlesByID (article_id) {
```

Comment:
One id refers to one article - so this should be selectArticleById, please!

Reply:
changed